pubspec.yaml

```diff
@@ -22,45 +22,49 @@ dependencies:
     sdk: flutter
 
   auto_route: 7.4.0
-  bloc: 8.1.1
+  bloc: ^8.1.2
```

Comment:
Let discuss it, because it can be a issue

Reply:
Here you have a good discussion to follow https://github.com/orgs/community/discussions/14200#discussioncomment-2601806